app/models/public_body.rb

```diff
@@ -216,6 +216,14 @@ def set_api_key!
     self.api_key = SecureRandom.base64(33)
   end
 
+  def self.find_by_name(name)
+    find_by(name: name)
+  end
+
+  def self.find_by_url_name(url_name)
```

Comment:
I think actually we can keep these – dynamic finders are [still supported](http://guides.rubyonrails.org/active_record_querying.html#dynamic-finders) so lets keep this commit as is?